src/main/java/com/microsoft/sqlserver/jdbc/DataTypes.java

```diff
@@ -795,7 +805,9 @@ enum SetterConversion {
 
         GEOMETRY(JDBCType.Category.GEOMETRY, EnumSet.of(JDBCType.Category.GEOMETRY)),
 
-        GEOGRAPHY(JDBCType.Category.GEOGRAPHY, EnumSet.of(JDBCType.Category.GEOGRAPHY));
+        GEOGRAPHY(JDBCType.Category.GEOGRAPHY, EnumSet.of(JDBCType.Category.GEOGRAPHY)),
+
+        VECTOR(JDBCType.Category.VECTOR, EnumSet.of(JDBCType.Category.VECTOR, JDBCType.Category.CHARACTER));
```

Comment:
(similar to above) Do we need JDBCType.Category.VECTOR in GetterConversion for the CHARACTER type, and vice versa?